lib/pbkit/pbkit.c

```diff
@@ -2353,32 +2353,32 @@ inline DWORD *pb_push4fto(DWORD subchannel, DWORD *p, DWORD command, float param
 
 inline DWORD *pb_push(DWORD *p, DWORD command, DWORD nparam)
 {
-    return pb_pushto(SUBCH_3D,p,command,nparam);
+    return pb_push_to(SUBCH_3D,p,command,nparam);
 }
 
 inline DWORD *pb_push1(DWORD *p, DWORD command, DWORD param1)
 {
-    return pb_push1to(SUBCH_3D,p,command,param1);
+    return pb_push1_to(SUBCH_3D,p,command,param1);
 }
 
 inline DWORD *pb_push2(DWORD *p, DWORD command, DWORD param1, DWORD param2)
 {
-    return pb_push2to(SUBCH_3D,p,command,param1,param2);
+    return pb_push2_to(SUBCH_3D,p,command,param1,param2);
 }
 
 inline DWORD *pb_push3(DWORD *p, DWORD command, DWORD param1, DWORD param2, DWORD param3)
 {
-    return pb_push3to(SUBCH_3Dp,command,param1,param2,param3);
```

Comment:
This had a bug in an earlier commit?

Reply:
That much is true. I'll try to sort it out in that earlier commit. :+1: